Header header
bool system_switch
bool motor_switch
bool auto_switch